### PR TITLE
added removeAccount

### PR DESCRIPTION
**Pull request by brunobar79**
Once https://github.com/MetaMask/eth-simple-keyring/pull/11 gets merged I can update the version of eth-simple-keyring here

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
     "browser-passworder": "^2.0.3",
     "eth-hd-keyring": "^1.2.2",
     "eth-sig-util": "^1.4.0",
-    "eth-simple-keyring": "^1.2.2",
+    "eth-simple-keyring": "github:brunobar79/eth-simple-keyring#remove-account",
```
Comment (danfinlay) [inline package.json]: Ready to be changed to `eth-simple-keyring@1.3.0`.

File: package.json [hunk 2]
```diff
@@ -38,7 +38,7 @@
     "browser-passworder": "^2.0.3",
     "eth-hd-keyring": "^1.2.2",
     "eth-sig-util": "^1.4.0",
-    "eth-simple-keyring": "^1.2.2",
+    "eth-simple-keyring": "^1.3.0",
```
Comment (danfinlay) [inline package.json]: Do we also need to include `eth-trezor-keyring@0.1.0`?

Comment (brunobar79) [reply package.json]: That's probably a good idea. Right now I'm adding the Keyring type in the metamask-controller (https://github.com/MetaMask/metamask-extension/pull/4625/files#diff-fd8e90928ae2cc670d306c0244a894d0R128)

If you're planning to add the multisig keyring here too then I should def. do it.

Comment (danfinlay) [reply package.json]: Oh right, it's included by consumers. That's fine with me.

**Pull request merged**